frontend/views/components/Menu/MenuContent.vue

```diff
@@ -1,19 +1,39 @@
 <template>
-  <div class="c-content" v-if="isActive" v-on-clickaway="closeMenu">
-    <slot></slot>
+  <div class="c-content" :class="{ 'is-active': isActive }">
+    <div v-if="isActive" v-on-clickaway="closeMenu" class="c-content-wrapper">
+      <slot></slot>
+    </div>
```

Comment:
Just noticed a bug when opening the ring bell. I believe some `position:absolute` or `width` is missplaced around :)

![image](https://user-images.githubusercontent.com/14869087/54295838-b6328880-45ab-11e9-9587-52e3f2a48daf.png)

Reply:
@pieer Oh, this MR was already closed 🤦‍♀️  If you want I can open an issue for that so it can be fixed later.

Reply:
Nice catch! I opened the issue: https://github.com/okTurtles/group-income-simple/issues/542 :)